src/components/cesium_map/cesiumh3.js

```diff
@@ -117,24 +100,63 @@ export class H3GridManager {
             console.log(`Grid ${rstr} already in collection`);
             return;
         }
-        if (rect !== GLOBAL_RECT) {
+        if (rect !== GLOBAL_RECT1) {
             this.global_grid = new H3Grid();
         }
         this.global_grid.rect_str = rstr;
-        // delete old grid
         let toRemove = cview.dataSources.getByName(this.old_grid_rstr)[0];
         cview.dataSources.remove(toRemove, true);
         const _this = this;
         this.global_grid.load(rstr).then((ds) => {
+            // delete existing grid
             try {
                 cview.dataSources.add(ds);
                 _this.current_grid = this.global_grid;
                 _this.current_rect = rect;
                 _this.old_grid_rstr = ds.name; // update 
+                let entities = ds.entities.values;
+                if (rstr === GLOBAL_RECT1 || rstr === GLOBAL_RECT2){
+                    //console.log("drawing on globe");
+                    for (let i = 0; i < entities.length; i++) {
+                        let entity = entities[i];
+                        try {
+                                let ln = parseFloat(entity.properties.ln);
+                                //let color = Cesium.Color.fromHsl(0.6 - ln * 0.5, 1.0, 0.5, 0.7);
```

Comment:
What is the commented out code?  Did you leave it in there on purpose?  We should probably get rid of it if not.

Reply:
Yes, I agree. Will remove the commented code!